binaries/conda/build_packages.py

```diff
@@ -6,17 +6,25 @@
 conda_build_dir = os.path.dirname(os.path.abspath(__file__))
 REPO_ROOT = os.path.join(conda_build_dir, "..", "..")
 MINICONDA_DOWNLOAD_URL = "https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh"
+CONDA_BINARY = f"$HOME/miniconda/condabin/conda"
 
 def install_conda_build():
     """
     Install conda-build, required to create conda packages
     """
-    os.system(f"conda install python=3.8 conda-build anaconda-client -y")
+    os.system(f"{CONDA_BINARY} install python=3.8 conda-build anaconda-client -y")
 
 def install_miniconda():
     """
     Installs miniconda, a slimmer anaconda installation to build conda packages
     """
+
+    # Check if conda binary already exists
+    exit_code = os.system(f"conda --version")
+    if exit_code == 0:
+        print(f"'conda' already present on the system. Proceeding without a fresh minconda installation.")
+        return
```

Comment:
CONDA_BINARY should be set if conda exists. Otherwise, it is possible the default CONDA_BINARY is different from existing conda path.

Reply:
Good point, thanks, fixed.